infrastructure/stacks/development-pipeline/codebuild.tf

```diff
@@ -204,7 +195,7 @@ resource "aws_codebuild_project" "di_deploy_dev" {
 
   logs_config {
     cloudwatch_logs {
-      group_name  = "/aws/codebuild/${var.project_id}-${var.environment}-deploy-${each.key}-stage"
+      group_name  = "/aws/codebuild/${var.project_id}-${var.environment}-deploy-stage"
```

Comment:
Is this line should be       
 group_name  = "/aws/codebuild/${var.project_id}-${var.environment}-deploy-**dev**-stage" ?

Reply:
This one should be dev as it can deploy the dev, test and perf environments.